src/astarte-charts/react/ConnectedDevices/index.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+   This file is part of Astarte.
+
+   Copyright 2020 Ispirata Srl
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+import React, { useMemo } from 'react';
+import { Pie } from 'react-chartjs-2';
+import type {
+  AstarteChartProvider,
+  AstarteChartSingleValue,
+  AstarteChartValueObject,
+} from 'astarte-charts';
+
+import { useChartProviders } from '../hooks';
+
+type ProviderValue = AstarteChartSingleValue<AstarteChartValueObject> | null;
+type Provider = AstarteChartProvider<ProviderValue>;
+
+const chartOptions = {
+  responsive: true,
+  legend: {
+    position: 'right',
+    align: 'start',
+  },
+};
+
+interface AstarteConnectedDevicesChartProps {
+  providers: [Provider];
+  height?: number;
+  width?: number;
+  refreshInterval?: number;
+}
+
+export const AstarteConnectedDevicesChart = ({
+  providers,
+  height,
+  width,
+  refreshInterval = 10000,
```

Comment:
After thinking about it, I changed it and left it disabled by default. So that:
- it doesn't require hitting the network more than once, if you don't specify otherwise
- we don't need to assume a specific interval is the preferred one